### DISCUSSION TITLE
Can we add something like this to versioning?

### DISCUSSION THREAD

**Discussion by berkantsahn** [squidfunk/mkdocs-material#5338] [Ideas]
A suggestion came to my mind for the versioning process. Maybe we can do something like this already, but I couldn't find an example.
So, for example, we have versions 1.0.0 and 2.0.0 in our project.

When I add a new file to my project in version 2.0.0, is there any chance of it being "new" icon next to it? Or can something like this be done? Or is it possible for me to have something like "update" icon next to it when I change a file?

**Comment by squidfunk** [comment]
You can [set the status for a page](https://squidfunk.github.io/mkdocs-material/reference/#setting-the-page-status) (currently sponsors only), which will show an icon next to it in the navigation:

``` markdown
---
status: new
---

# Headline
...
```

Our [tooltips section](https://squidfunk.github.io/mkdocs-material/reference/tooltips/) currently has it + it shows a custom overlay when tooltips are enabled:

<img width="171" alt="Bildschirm­foto 2023-04-07 um 16 25 09" src="https://user-images.githubusercontent.com/932156/230625292-43d96a8b-1f15-41c7-8b58-94bab8bf7f8f.png">

If you want to do this automatically during deployment, I suggest looking into [hooks](https://www.mkdocs.org/user-guide/configuration/#hooks), which allow you to alter pages or configuration before building. You would need to write some Python for that, but it's rather straight forward.

**Reply by berkantsahn** [reply]
Thanks. I am appreciate.

**Reply by squidfunk** [reply]
This was double posted https://github.com/mkdocs/mkdocs/discussions/3174 – please don't do that, you're duplicating efforts...

**Reply by berkantsahn** [reply]
I was thinking that the people interested in the repo were a different group. That's why I opened a post in two different places. Thanks for warning.

**Reply by squidfunk** [reply]
Fair enough then. Then I would kindly ask you to link to the other post next time, so we know it right from the start – it's more than often the time of maintainers that goes into this that is then missing for bugfixes or feature requests. The reaosn is that we can't always check the discussion board and issue trackers of all related dependencies before answering, so we need to rely on the cooperation of our users. Thanks for your understanding! ☺️
